src/module.c

```diff
@@ -12538,6 +12522,38 @@ int moduleLoad(const char *path, void **module_argv, int module_argc, int is_loa
     return C_OK;
 }
 
+/* Load a module by path and initialize it. On success C_OK is returned, otherwise
+ * C_ERR is returned. */
+int moduleLoad(const char *path, void **module_argv, int module_argc, int is_loadex) {
```

Comment:
can you move this to above moduleOnLoad? it'll do better to the blame log / conflicts.

Reply:
done with [`c3fdca8` (#13915)](https://github.com/redis/redis/pull/13915/commits/c3fdca8acc827d6349c5d51d05a7b5072040f6e5)

Reply:
i don't see it yet in the PR. 
p.s. it would be nice if we amended the commit that split it to be in the right order (and force pushed), since we won't squash-merge this PR.
not mandatory, since regardless, git will handle merge conflicts correctly (even if the blame log is broken).